src/command/template/templates.rs

```diff
@@ -0,0 +1,111 @@
+use crate::command::template::GithubTemplate;
+use crate::options::ProjectLanguage;
+use crate::{anyhow, error::RoverError, Result, Suggestion};
+
+use std::collections::HashMap;
+use std::iter::IntoIterator;
+
+use console::Term;
+use dialoguer::Select;
+
+/// TODO: Fetch templates from an API instead of embedding them
+const TEMPLATES: [GithubTemplate; 4] = [
+    GithubTemplate {
+        id: "subgraph-javascript-apollo-server",
+        git_url: "https://github.com/apollographql/subgraph-template-javascript-apollo-server-boilerplate",
+        display: "Apollo Server",
+        language: ProjectLanguage::Javascript,
+    },
+    GithubTemplate {
+        id: "subgraph-python-strawberry-fastapi",
+        git_url: "https://github.com/strawberry-graphql/subgraph-template-strawberry-fastapi",
+        display: "Strawberry with FastAPI",
+        language: ProjectLanguage::Python,
+    },
+    GithubTemplate {
+        id: "subgraph-python-ariadne-fastapi",
+        git_url: "https://github.com/patrick91/subgraph-template-ariadne-fastapi",
+        display: "Ariadne with FastAPI",
+        language: ProjectLanguage::Python,
+    },
+    GithubTemplate {
+        id: "subgraph-rust-async-graphql",
+        git_url: "https://github.com/apollographql/subgraph-template-rust-async-graphql",
+        display: "async-graphql with Axum",
+        language: ProjectLanguage::Rust,
+    }
+];
+
+pub struct GithubTemplates {
+    templates: HashMap<&'static str, GithubTemplate>,
```

Comment:
I have a suggestion to reduce overhead which I'm putting in a PR to this PR since suggestions are hard 😁.

I like having this struct, though. This should make additional filters easier in the future and we can sub out the hard-coded init for an API query after filters are applied in the future.

Reply:
yeah we could maybe just keep this a `Vec` like you had it 🤷🏼